lib/MusicBrainz/Server/Edit/Release/ReorderMediums.pm

```diff
@@ -1,4 +1,5 @@
 package MusicBrainz::Server::Edit::Release::ReorderMediums;
+use utf8;
```

Comment:
If I remember correctly, this is more like `from __future__ import unicode_literals` combined with `# -*- coding: utf-8 -*-`.